less/matching.less

```diff
@@ -11,6 +11,10 @@
 	.matching-select-container {
 		position: relative;
 		background-color: @item-color;
+
+		&.error {
+			border: 1px solid @validation-error;
```

Comment:
If we use `outline` instead then there wouldn't be a slight size shift on the page when the class is applied.